tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/null_invalid_foreign_keys_test.py

```diff
@@ -179,16 +179,10 @@ def test_get_query_specs(self, mock_get_affected_tables,
                 nifk.SANDBOX_QUERY.render(
                     project_id=self.project_id,
                     sandbox_dataset_id=self.sandbox_id,
-                    intermediary_table='dc-388_dc-807_person',
+                    intermediary_table='dc388_dc807_person',
```

Comment:
We can just use the intermediary table with one ticket number. having 2 ticket numbers in the table name will be confusing.